src/components/studyBrowser/ThumbnailEntry.js

```diff
@@ -18,6 +18,7 @@ class ThumbnailEntry extends Component {
     error: PropTypes.bool.isRequired,
     active: PropTypes.bool.isRequired,
     stackPercentComplete: PropTypes.number,
+    modality: PropTypes.string,
```

Comment:
Hmm.. Would we ever want this same functionality for encapsulated PDFs or other file types? I'd maybe change this to `altImageText` or something similar so it travels a bit further. Then, add a quick prop comment to indicate it will only be rendered/used if imageSrc is null/undefined

Reply:
done